Sources/CoalModule/Verification/VerificationCodeView.swift

```diff
@@ -29,50 +29,64 @@ public struct VerificationCodeView: View {
       pageType: .verificationCode,
       leftAction: { navigator?.popToPreviousView() },
       backgroundImage: backgroundImage,
-      backgroundColor: backgroundColor
+      backgroundColor: backgroundColor,
+      isShowNavBar: false
     ) {
       VStack(spacing: 20) {
+        Spacer()
         bottomSheetView
       }
     }
   }
   
   private var bottomSheetView: some View {
     BottomSheetView {
-      AuthenticationHeaderView(
-        configHeader: config?.verificationCodeHeader,
-        additionalParam: (sendTo ?? config?.sendVerificationCodeTo) ?? "",
-        alignment: .center
-      ).padding(.top, 30)
-      
-      CodeFieldView(
-        code: $viewModel.code,
-        isError: viewModel.isError,
-        isTimerActive: viewModel.isTimerActive,
-        remainingTime: viewModel.remainingTime,
-        onResetError: {
-          viewModel.clearError()
-        },
-        onResendCode: {
-          viewModel.resetTimer()
-        })
-      
-      CoalButtonView(
-        field:
-          ConfigField(
-            type: .submit,
-            label: CoalString.verify
-          ),
-        isDisabled: !viewModel.isOTPComplete || viewModel.isError
-      ) {
-        if viewModel.verifyOTP(correctOTP: viewModel.correctOTP) {
-          navigator?.showVerificationMethodPage()
+      VStack(spacing: 0) {
+        CoalNavBar(
```

Comment:
navbar didalem aja baseview ki, jadi as is aja. ga perlu ikut figma. karena behaviournya beda. komponenya jadi numpuk gak clean.